src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl

```diff
@@ -0,0 +1,12 @@
+task quay {
+    command {
+        echo "hello"
+    }
+    runtime {
+        docker: "quay.io/tjeandet/tj-ubuntu:centaur"
```

Comment:
I can't find an "official repositories" equivalent of dockerhub for quay but http://www.appcontainers.com/ have an ubuntu repo on quay so I can use that instead